src/server/system_services/bucket_server.js

```diff
@@ -129,13 +131,19 @@ async function create_bucket(req) {
         changes.insert.tiers = [tier];
     }
 
+    const bucket_m_key = master_key_manager.get_instance().new_master_key({
+        description: 'Bucket master key',
```

Comment:
`description` should somehow explain the key, we shouldn't just have the same description to everyone.
Please check in all descriptions (system, bucket, account etc...) and maybe add some uniqueness to them.

Reply:
added ids for the uniqueness